API.md

```diff
@@ -0,0 +1,66 @@
+# Octofet API
+
+## `class Octofet`
+
+Create an object of type `Octofet` to communicate with a particular [Octofet board](https://my.amperka.com/modules/octofet).
+
+### `Octofet(uint8_t pinCS, uint8_t devicesCount = 1, uint8_t* resetValues = NULL)`
```

Comment:
The constructor takes a `resetValues` array, and the deviceWrite function takes an 8-bit `value`. I think uniformity is needed in prototypes.